pkg/controller/util.go

```diff
@@ -27,6 +28,13 @@ const DataVolName = "cdi-data-vol"
 const ImagePathName = "image-path"
 const socketPathName = "socket-path"
 
+type podDeleteRequest struct {
```

Comment:
I set up a request here because we're still a bit disjointed in our controllers.  In the future, we'd ideally do something like an interface and share the bulk of the Controller functions, but for now I'm cheating and use a request struct so you can shove in all the nitty gritty details.